ci/docker/install/deb_ubuntu_ccache.sh

```diff
@@ -50,15 +51,15 @@ export PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
 
 git clone --recursive https://github.com/ccache/ccache.git
 cd ccache
-# Checkout a fixed & tested pre-release commit of ccache 4
-# ccache 4 contains fixes for caching nvcc output: https://github.com/ccache/ccache/pull/381
-git checkout 2e7154e67a5dd56852dae29d4c418d4ddc07c230
+git checkout v3.7.8
+# Backport cuda related fixes: https://github.com/ccache/ccache/pull/381
+git config user.name "MXNet CI"
```

Comment:
Can you do that at the place where git is installed?

Reply:
No. I'm setting this only for the ccache repository.

Reply:
I'd suggest having a patched fork somewhere and we can pick up ccache from that instead.

Reply:
That doesn't differ substantially from the current practice (this PR) but will introduce a delay in fixing the problem in the order of 12-18 hours, as any change needs to pass CI again.

Reply:
Any commiter has the permission to modify this PR. I don't mind any changes. Just the person modifying it should make sure to take 8 hours after change to retrigger the CI as often as needed.